charts/aws-event-sources/0.1.0/Chart.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v2
```

Comment:
This will need to be changed to `v1` for Helm 2 compatibility since is currently causing this chart to fail the install in Rancher.